src/components/Menu/MenuItem.tsx

```diff
@@ -144,6 +146,7 @@ class MenuItem extends AutoControlledComponent<ReactProps<MenuItemProps>, MenuIt
     defaultMenuOpen: PropTypes.bool,
     onActiveChanged: PropTypes.func,
     inSubmenu: PropTypes.bool,
+    kind: PropTypes.string,
```

Comment:
```suggestion
    kind: PropTypes.oneOf(['divider']),
```

Reply:
Why should the type here be `oneOf` containing divider? MenuItem will actually never be created if the kind prop is divider. For the menu item it can be 'item' according to the convention, or nothing as this is the default component. Am I wrong assuming this?

Reply:
So, `kind: PropTypes.oneOf(['divider', 'item']),`? We should disallow to pass invalid values to `kind`

Reply:
Resolved in personal conversation :)